Src/Notion.Client/Extensions/QualityOfLifeExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Notion.Client.Extensions
+{
+    public static class QualityOfLifeExtensions
```

Comment:
@Kurupapuru I'm sure there could be many more helpful extensions we may want to add. So I think it is better if we create a separate library (NuGet) to maintain them. So the user can inject them as they require. Let me know your views?

Reply:
@KoditkarVedant sorry for such late response, expected to get notification from github
I think that is a good option. I don't have any experience with creating projects for publishing nuget packages, but if you will create repository with needed structure I can make pull request there.

Reply:
@Kurupapuru I've created a repository so you can add these extensions there.

**Repo:**  https://github.com/notion-dotnet/notion-sdk-extensions